src/main/java/com/ecommerce/GPDemo/controller/UserController.java

```diff
@@ -1,29 +1,50 @@
 package com.ecommerce.GPDemo.controller;
 
-import com.ecommerce.GPDemo.entity.login;
+import com.ecommerce.GPDemo.entity.Login;
 import com.ecommerce.GPDemo.service.UserService;
-import com.ecommerce.GPDemo.Response.AuthenticationResponse;
+import com.ecommerce.GPDemo.response.AuthenticationResponse;
+
 import com.ecommerce.GPDemo.dto.UserDTO;
-import com.ecommerce.GPDemo.Response.RegisterResponse;
+import com.ecommerce.GPDemo.response.RegisterResponse;
+import com.fruits.ecommerce.api.UserApi;
+import org.modelmapper.ModelMapper;
 import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.Map;
 
 @RestController
 @CrossOrigin
 @RequestMapping("api/v1/user")
-public class UserController {
+public class UserController implements UserApi{
     @Autowired
     private UserService userService;
 
-    @PostMapping("/register")
-    public RegisterResponse addUser(@RequestBody UserDTO userDTO){
-        return userService.addUser(userDTO);
-    }
+    @Autowired
+    private ModelMapper modelMapper;
 
+//    @PostMapping("/register")
+//    public RegisterResponse addUser(@RequestBody UserDTO userDTO){
+//        return userService.addUser(userDTO);
+//    }
+//
+//    @PostMapping("/login")
+//    public AuthenticationResponse loginUser(@RequestBody Login log, @RequestParam Map<String , Object> claims)  {
+//        return userService.loginUser(log);
+//    }
+
+
+    @Override
     @PostMapping("/login")
-    public AuthenticationResponse loginUser(@RequestBody login log, @RequestParam Map<String , Object> claims)  {
-        return userService.loginUser(log);
+    public ResponseEntity<com.fruits.ecommerce.model.AuthenticationResponse> _loginUser(@RequestBody com.fruits.ecommerce.model.Login login) {
```

Comment:
No need for the full path "com.fruits.ecommerce.model.AuthenticationResponse", just import it

Reply:
you have to override method loginUser not _loginUser